.TOOLVERSIONS

```diff
@@ -1,4 +1,6 @@
 android-sdk-build-tools;28.0.1;
 android-sdk-platform;android-27;
 android-sdk;4333796;aa190cfd7299cd6a1c687355bb2764e4
+node;10.12.0;
```

Comment:
ok, since you were callling toolversions from the node2nix script, I thought it got missing. So the nix branch should be compiled against node 8.x for now then?

Reply:
what yarn expression are you refering to?

Reply:
> ok, since you were callling toolversions from the node2nix script, I thought it got missing. So the nix branch should be compiled against node 8.x for now then?

oh right, that was a mistake, I shouldn't have removed `react_native_cli` from `.TOOLVERSIONS`. I'll fixup the commit.

Reply:
> what yarn expression are you refering to?

`yarn` here is based on 8.x: https://github.com/status-im/status-react/blob/feature%2Fnix/default.nix#L34. Initially I was using node 10.x until I realized that it was causing builds to fail due a mismatch in node versions.